src/facilities/index.js

```diff
@@ -235,12 +256,55 @@ class Facilities {
   getAllByOrganizationId(organizationId) {
     if (!organizationId) {
       return Promise.reject(
-        new Error("An organization id is required for getting a list of an organization's facilities")
+        new Error(
+          "An organization id is required for getting a list of an organization's facilities"
+        )
+      );
+    }
+
+    return this._request
+      .get(`${this._baseUrl}/organizations/${organizationId}/facilities`)
+      .then(facilities =>
+        facilities.map(facility => formatFacilityFromServer(facility))
+      );
+  }
+
+  /**
+   * Gets a list of all facilities that belong to a particular organization.
+   * Also, includes the groupings that each facility belongs to.
+   *
+   * API Endpoint: '/organizations/:organizationId/facilities?includeGroupings=true'
+   * Method: GET
+   *
+   * @param {string} organizationId UUID corresponding with an organization
+   *
+   * @returns {Promise}
+   * @fulfill {Facility[]} Information about all facilities
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.facilities.getAllByOrganizationIdWithGroupings(25)
+   *   .then((facilities) => console.log(facilities));
+   *   .catch((err) => console.log(err));
+   */
+  getAllByOrganizationIdWithGroupings(organizationId) {
+    if (!organizationId) {
+      return Promise.reject(
+        new Error(
+          "An organization id is required for getting a list of an organization's facilities"
+        )
       );
     }
 
-    return this._request.get(`${this._baseUrl}/organizations/${organizationId}/facilities`)
-      .then((facilities) => facilities.map((facility) => formatFacilityFromServer(facility)));
+    return this._request
+      .get(
```

Comment:
It's preferable to pass the params as an object since they are way easier to maintain that way
```
    return this._request
      .get(`${this._baseUrl}/organizations/${organizationId}/facilities`, {
        params: { includeGroupings: true }
      })
      .then(facilities =>
        facilities.map(facility => formatFacilityFromServer(facility))
      );
```